ManUp/ManUp.m

```diff
@@ -265,10 +282,31 @@ - (void)checkAppVersion {
     NSComparisonResult minVersionComparisonResult = [ManUp compareVersion:installedVersion toVersion:minVersion];
     NSComparisonResult currentVersionComparisonResult = [ManUp compareVersion:installedVersion toVersion:currentVersion];
     
-    if (self.alertController) {
+    if (self.alertController && !self.maintenanceModeShown) {
         [self log:@"ManUp: An alert is already displayed, aborting."];
         
     } else {
+        BOOL enabled = [self settingForKey:kManUpConfigAppIsEnabled] ? [[self settingForKey:kManUpConfigAppIsEnabled] boolValue] : YES;
+        if (enabled && self.maintenanceModeShown) {
+            self.maintenanceModeShown = NO;
+            [self.alertController dismissViewControllerAnimated:YES completion:nil];
+            self.alertController = nil;
+            
+        } else if (!enabled && !self.maintenanceModeShown) {
+            [self showAlertWithTitle:[NSString stringWithFormat:NSLocalizedString(@"%@ Unavailable", nil), appName]
+                             message:[NSString stringWithFormat:NSLocalizedString(@"%@ is currently unavailable. Please check back later.", nil), appName]
```

Comment:
Add comments to localized strings where parameters are given eg "{appName} Unavailable"

Reply:
Will fix 💃